Automattic/wp-calypso
# Pricing page: update product icon color and design

#### Proposed Changes

* Update included products color into lighter variant.

   **Before**
<img width="1107" alt="Screen Shot 2022-12-02 at 8 46 31 PM" src="https://user-images.githubusercontent.com/56598660/205296143-4735a44c-1caa-45fc-8fb7-1311f948238e.png">

   **After**
   <img width="1097" alt="Screen Shot 2022-12-02 at 8 41 48 PM" src="https://user-images.githubusercontent.com/56598660/205295449-38486c16-b473-4a41-91b7-f2df6b42023b.png">

* Update product icons in the Product section with new designed icons.
<img width="1320" alt="Screen Shot 2022-12-02 at 8 44 03 PM" src="https://user-images.githubusercontent.com/56598660/205295769-0adcab59-4a29-4c11-865c-c18c3225b04c.png">



#### Testing Instructions

1. Run this branch by following the steps below (or you can use the Jetpack Cloud live link commented on this PR)
    * Run `git fetch && git checkout update/pricing-page-included-products-icon-to-dark`
    * Run `yarn start-jetpack-cloud`

2. Click on `More about Complete` or `More about Security` links to show the bundled lightbox.
3. Confirm that the included products icon renders a light version.
4. Close the lightbox and click `Products` filter button.
5. Confirm that the product icons is now updated with the new design.

#### Pre-merge Checklist





Related to 1202858161751496-as-1203463248826782
